spec/post_integraion_spec.rb

```diff
@@ -0,0 +1,123 @@
+require 'rails_helper'
+
+RSpec.configure do |config|
+  config.before(:each, type: :system) do
+    driven_by :selenium, using: :chrome, options: {
+      binary: '/usr/bin/google-chrome' # Replace with the actual Chrome binary location on your Chromebook
+    }
+  end
+end
+
+RSpec.describe 'Post', type: :system do
+  before do
+    driven_by(:rack_test)
+  end
+
+  let(:user1) do
+    User.create(name: 'John', photo: 'https://media.giphy.com/media/tZaFa1m8UfzXy/giphy.gif',
+                bio: 'This is John\'s bio', posts_counter: 0)
```

Comment:
- [ ] on your `post#index` tests, Kindly add :
  - Tests that asserts you can see a section for pagination if there are more posts than fit on the view.